section4/reflection.md

```diff
@@ -1,22 +1,37 @@
 ## Section 4 Reflection
 
 1. How different did your workflow feel this week, considering we asked you to follow the Pomodoro technique?
-
+  * I was able to work for a longer period of time over the course of the day. I normally don't take breaks and stop working
+    whenever I get tired. However, when using the Pomodoro technique, I forced myself to take breaks and was able to work longer over the course of a day in a more concentrated state.  
 1. Regarding the work you did around setting intentions in Step 1 of the Pomodoro technique - how did that go? Were you surprised by anything (did you find yourself way more focused than you realized, more distracted that you thought you'd be, estimating times accurately or totally off, etc)?
-
+  * Estimating time is still a work in progress because it is dependent on how long it takes me to learn/feel comfortable with the material. When learning classes and objects at the beginning I was confused so I slowed down and made sure I understood exactly what was going on each section before progressing. With the Pomodoro Technique I was able to relax and focus on what I was learning because I knew no matter what, I was going to be studying for a certain amount of time.
```

Comment:
Estimating time is HARD, and even experienced engineers joke about how it's impossible. So be nice to yourself, and continue to use those pomodoros and document what you get done so you can continue to increase your awareness of where times goes. And also, it's hard.